test/kong-tests-compose.yaml

```diff
@@ -0,0 +1,90 @@
+version: '2.1'
```

Comment:
at some point I think merging docker-kong or making it a submodule or doing something to reduce copy-pasta will be nice. Also would make building nightlies easier.

Nothing related to this PR just talking out loud

Reply:
This is a slightly modified version of the docker-kong compose - without the Docker volume definition (making sure each test spinning up a Kong gets a clean env) plus some extra env vars.